examples/tutorials/WebsocketClientWorker.ipynb

```diff
@@ -0,0 +1,762 @@
+        "\n",
+        "In that tutorial, we'll demonstrate how to launch a WebsocketWorker server which will listen for PyTorch commands over a socket connection. In this tutorial, the two workers are connected via a socket connection on the localhost network.\n",
+        "\n",
+        "If you'd prefer to download this notebook and run it locally, you can do so via the `Download .ipynb` in the `File` dropdown field in this Google colab environment. \n",
+        "\n",
+        "Setup instructions: https://research.google.com/colaboratory/local-runtimes.html"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LkAOFaq3WdCn",
+        "colab_type": "text"
+      },
+      "source": [
+        "\n",
+        "# Step -1: Copy This Notebook\n",
+        "\n",
+        "Go up to File -> Save A Copy in Drive\n",
+        "\n",
+        "This will let you execute the notebook (it won't let you execute this one by default)\n",
+        "\n",
+        "# Step 0: Install Dependencies"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "G58GV77oWgO-",
+        "colab_type": "code",
+        "outputId": "b6080406-3e39-47b6-e39c-95709cd8d932",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 5154
+        }
+      },
+      "source": [
+        "# If not using local runtimes, uncomment out the following snippets where necessary.\n",
+        "\n",
+        "#! rm -rf /content/PySyft\n",
+        "#! git clone https://github.com/OpenMined/PySyft.git\n",
+        "# http://pytorch.org/\n",
+        "#from os import path\n",
+        "#from wheel.pep425tags import get_abbr_impl, get_impl_ver, get_abi_tag\n",
+        "\n",
+        "#platform = '{}{}-{}'.format(get_abbr_impl(), get_impl_ver(), get_abi_tag())\n",
+        "#cuda_output = !ldconfig -p|grep cudart.so|sed -e 's/.*\\.\\([0-9]*\\)\\.\\([0-9]*\\)$/cu\\1\\2/'\n",
+        "#accelerator = cuda_output[0] if path.exists('/opt/bin/nvidia-smi') else 'cpu'\n",
+        "\n",
+        "#!pip3 install https://download.pytorch.org/whl/cu100/torch-1.1.0-cp36-cp36m-linux_x86_64.whl\n",
+        "#!pip3 install https://download.pytorch.org/whl/cu100/torchvision-0.3.0-cp36-cp36m-linux_x86_64.whl\n",
+        " \n",
+        "import torch\n",
+        "\n",
+        "#!cd PySyft; pip3 install -r requirements.txt; pip3 install -r requirements_dev.txt; python3 setup.py install\n",
+        "\n",
+        "import os\n",
+        "import sys\n",
+        "\n",
+        "#module_path = '/content/PySyft'  # You want './PySyft' to be on your sys.path\n",
+        "#if module_path not in sys.path:\n",
+        "#    sys.path.append(module_path)"
+      ],
+      "execution_count": 0,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "fatal: destination path 'PySyft' already exists and is not an empty directory.\n",
```

Comment:
Can you clean the output?

Reply:
I will clean the output but it is [copied straight from this example from openmined.org ](https://colab.research.google.com/drive/1Je1rk7olA9uTWWaqvvt4_gXf7yX1rTBm) which also includes this lines. May the example there should also be fixed.

Reply:
Or is this output produced as soon as i open it with my colab?

Reply:
I have cleard the ouputs and also moved the Websocket Tutorials to a own folder